### PR TITLE
Fixing type (missing function call) in example for hook_civicrm_alterAngular

### PR DESCRIPTION
**Pull request by michaelmcandrew**
I followed the example code. It didn't work. I found [another example that did work](https://github.com/civicrm/civicrm-core/pull/10085), so I am updating this example.

### CODE REVIEW

**Comment by seamuslee001** [comment]
@michaelmcandrew where does $changeset come from ? on the whole this seems good

**Comment by michaelmcandrew** [comment]
Thanks @seamuslee001 - well spotted! That was something else I tripped over but had forgotten to include. I've added the namespace now.

**Comment by michaelmcandrew** [comment]
PS turns out the missing function call wasn't as missing as I thought it was :s

Still, this is an improvement

File: CRM/Utils/Hook.php [hunk 1]
```diff
@@ -2133,11 +2133,12 @@ public static function angularModules(&$angularModules) {
    *
    * @code
    * function example_civicrm_alterAngular($angular) {
-   *   $angular->add(ChangeSet::create('mychanges')
+   *   $changeSet = \Civi\Angular\ChangeSetChangeSet::create('mychanges')
```
Comment (seamuslee001) [inline CRM/Utils/Hook.php]: @michaelmcandrew you got too many ChangeSet i think it should be `\Civi\Angular\ChangeSet::create`

Comment (eileenmcnaughton) [reply CRM/Utils/Hook.php]: Ch-ch-ch-ch-changes.... sounds like a song...

**Comment by michaelmcandrew** [comment]
I watch the ripples change their size, but never leave the stream...

**Comment by seamuslee001** [comment]
@eileenmcnaughton @totten this looks fine to be merged now

**Comment by totten** [comment]
Nice catch @michaelmcandrew .

**Pull request merged**